js_modules/dagster-ui/packages/ui-core/src/assets/__fixtures__/AssetViewDefinition.fixtures.ts

```diff
@@ -145,3 +152,35 @@ export const AssetViewDefinitionSDA = buildQueryMock<
     }),
   },
 });
+
+export const RootWorkspaceWithOneLocation = buildQueryMock<
+  RootWorkspaceQuery,
+  RootWorkspaceQueryVariables
+>({
+  query: ROOT_WORKSPACE_QUERY,
+  variables: {},
+  data: {
+    workspaceOrError: buildWorkspace({
+      locationEntries: [
+        buildWorkspaceLocationEntry({
+          id: 'ipsum-entry',
+          name: 'ipsum-entry',
+          locationOrLoadError: buildRepositoryLocation({
+            id: 'ipsum',
+            name: 'ipsum',
```

Comment:
The builders put mock data in already so you don't need to list out every field.